test/e2e/tests/validator_test.go

```diff
@@ -77,11 +77,8 @@ func TestValidator_Propose(t *testing.T) {
 
 		require.False(t, proposeCount == 0 && expectCount > 0,
 			"node did not propose any blocks (expected %v)", expectCount)
-		if expectCount > 3 {
+		if expectCount > 5 {
 			require.GreaterOrEqual(t, proposeCount, 3, "validator didn't propose even 3 blocks")
```

Comment:
Does this line need to change too?

Reply:
No. What we're saying is that if we expected it to propose 6 (or more times), we're fine with 3 -- less than 6, we're fine with 1. The previous failure was because we required 3 out of 3, but it only proposed 2.

Reply:
Ohhh, I see. Alright!